app/components/primer/alpha/segmented_control.html.erb

```diff
@@ -0,0 +1,12 @@
+<segmented-control>
+  <%= render Primer::BaseComponent.new(tag: :ul, role: "list", **@system_arguments) do %>
+    <% buttons.each do |button| %>
+      <li class="<%= class_names(
```

Comment:
Does it make sense for this `<li>` to be emitted by the `Button` instead of here?